src/screens/CreateWallet/screens/CreateWalletStep2/index.tsx

```diff
@@ -105,19 +105,22 @@ export const CreateWalletStep2 = () => {
     );
 
     const onPressOnSelected = () => {
-      if (selectedIdx > -1) {
-        walletMnemonicSelected.splice(selectedIdx, 1);
-      } else {
-        walletMnemonicSelected.push(word);
+      if (word.word) {
```

Comment:
try to rewrite these flow word.word, seems strange, maybe try to destruct these object

Reply:
logic changed